diff --git a/Wikipedia-iOS/Plugins/ReadItLater/ReadItLaterLite.m b/Wikipedia-iOS/Plugins/ReadItLater/ReadItLaterLite.m
index b02a1cf..ab70e4d 100644
--- a/Wikipedia-iOS/Plugins/ReadItLater/ReadItLaterLite.m
+++ b/Wikipedia-iOS/Plugins/ReadItLater/ReadItLaterLite.m
@@ -12,7 +12,7 @@
 
 @implementation ReadItLater
 
-static NSString *apikey = @"[RIL-API-KEY]";			//Enter your apikey here : get one at http://readitlaterlist.com/api/ 
+static NSString *apikey = @"[YOUR-RIL-API-KEY]";			//Enter your apikey here : get one at http://readitlaterlist.com/api/
 static NSString *nameOfYourApp = @"Wikipedia";		//Enter the name of your application here (optional - for user-agent string)
 
 
diff --git a/Wikipedia-iOS/Wikipedia-iOS-Info.plist b/Wikipedia-iOS/Wikipedia-iOS-Info.plist
index 7b5b642..7394dfd 100644
--- a/Wikipedia-iOS/Wikipedia-iOS-Info.plist
+++ b/Wikipedia-iOS/Wikipedia-iOS-Info.plist
@@ -74,7 +74,7 @@
             <string>org.wikipedia.fb</string>
             <key>CFBundleURLSchemes</key>
             <array>
-				<string>fb[FB-APP-ID]</string>
+				<string>fb[YOUR-FB-APP-ID]</string>
             </array>
         </dict>
     </array>
diff --git a/assets/www/ios/platform.js b/assets/www/ios/platform.js
index 88245d9..43e0a3d 100644
--- a/assets/www/ios/platform.js
+++ b/assets/www/ios/platform.js
@@ -67,7 +67,7 @@ function shareRIL() {
 
 chrome.addPlatformInitializer(function() {
 	console.log("Logging in!");
-	window.plugins.FB.init("[FB-APP-ID]", function() {
+	window.plugins.FB.init("[YOUR-FB-APP-ID]", function() {
 		console.log("failed FB init:(");
 	});
 	console.log("Logged in!");
