diff --git a/javascripts/references.js b/javascripts/references.js
index 6e212e5..1b0714d 100644
--- a/javascripts/references.js
+++ b/javascripts/references.js
@@ -123,7 +123,7 @@ if( typeof jQuery !== 'undefined' ) {
 				});
 			}
 		}
-		init( $("#content")[0], true );
+		//init( $("#content")[0], true );
 		return {
 			init: init
 		};
diff --git a/javascripts/toggle.js b/javascripts/toggle.js
index 9c5c9fd..0e70826 100644
--- a/javascripts/toggle.js
+++ b/javascripts/toggle.js
@@ -115,7 +115,7 @@ MobileFrontend.toggle = (function() {
 		}, 400); // this matches transition speed in sections.css
 	}
 
-	init();
+	//init();
 	return {
 		wm_reveal_for_hash: wm_reveal_for_hash,
 		wm_toggle_section: wm_toggle_section,
