diff --git a/javascripts/references.js b/javascripts/references.js
index 32d4206..d404059 100644
--- a/javascripts/references.js
+++ b/javascripts/references.js
@@ -43,7 +43,7 @@ if( typeof jQuery !== 'undefined' ) {
 		function init( container, firstRun ) {
 			var el, close, lastLink, data, html, href, references = collect();
 			container = container || $("#content")[0];
-			firstRun = typeof( firstRun ) === 'undefined' ? true : false;
+			firstRun = typeof( firstRun ) === 'undefined' ? true : firstRun;
 			$("#mf-references").remove();
 			el = $( '<div id="mf-references"><div></div></div>' ).hide().
 				appendTo( document.body )[0];
